src/js/components/cycleCount/toResolveTab/ResolveStepHeader.jsx

```diff
@@ -74,6 +74,7 @@ const ResolveStepHeader = ({
           label="react.default.button.next.label"
           defaultLabel="Next"
           variant="primary"
+          //disabled={disableNextButton()}
```

Comment:
@EWaterman I think this should be removed and we don't need the `disableNextButton` anymore, but I left that for you to see, for the clarity.